src/backend/distributed/connection/connection_management.c

```diff
@@ -409,6 +409,26 @@ CloseConnectionByPGconn(PGconn *pqConn)
 
 
 /*
+ * FinishConnectionListEstablishment is a wrapper around FinishConnectionEstablishment.
+ * The function iterates over the multiConnectionList and finishes the connection
+ * establishment for each multi connection.
+ */
+void
+FinishConnectionListEstablishment(List *multiConnectionList)
```

Comment:
For the future: I'd *much* prefer such functionally independent parts of a commit to be done independently.